### DISCUSSION TITLE
Clarification on gene conversion - chromosome ends?

### DISCUSSION THREAD

**Discussion by csmcal** [tskit-dev/msprime#1661] [Q&A] [answered]
A question on how gene conversion works - are gene conversion events that include tracts that pass independence points - a recombination probability of 0.5 between positions  - thrown out? Or do the tracts go through them?

**Comment by jeromekelleher** [comment]
Sorry for the delay in getting back to you @csmcal. I think the GC and recombination processes are basically independent, and we're definitely not taking into account independence points like you mention. So yes, there is an issue with our recipe for simulating multiple chromosomes if you want to also include GC.

@fbaumdicker, is this correct?

**Reply by csmcal** [reply]
Thanks for the clarification!

**Comment by fbaumdicker** [comment, marked as answer]
Sorry, I missed this question so far.
Yes, gene conversion is basically independent of the recombination process.
We did not think of independence points, as you could just simulate multiple chromosomes in independent simulation runs.
However, if you would like to simulate multiple chromosomes at once and include independence points as well as gene conversion, one solution might be to add a long enough dummy sequence around the independence points, such that it is extremely unlikely to span the complete dummy sequence in a GC event.

**Reply by csmcal** [reply]
Thanks for the suggestion! That should work fine, I'm already padding the ends
